includes/class-wc-coupon.php

```diff
@@ -994,6 +996,17 @@ public function get_coupon_error( $err_code ) {
 			case self::E_WC_COUPON_NOT_APPLICABLE:
 				$err = __( 'Sorry, this coupon is not applicable to your cart contents.', 'woocommerce' );
 				break;
+			case self::E_WC_COUPON_USAGE_LIMIT_COUPON_STUCK:
+				if ( is_user_logged_in() && wc_get_page_id( 'myaccount' ) > 0 ) {
+					// translators: %s: myaccount page link.
```

Comment:
For translator comments, I think we generally use `/* */` comment format.

Reply:
thanks, fixed in https://github.com/woocommerce/woocommerce/pull/28592/commits/f248276a89d104db3258bc139f663393bbace98b